src/System.Composition.Convention/src/System/Composition/Convention/ConventionBuilder.cs

```diff
@@ -288,8 +288,8 @@ private List<Attribute> ReadMemberCustomAttributes(Type reflectedType, System.Re
         /// <returns>The list of applied attributes.</returns>
         public override IEnumerable<Attribute> GetCustomAttributes(Type reflectedType, System.Reflection.ParameterInfo parameter)
         {
-            Requires.NotNull(parameter, "reflectedType");
-            Requires.NotNull(parameter, "parameter");
+            Requires.NotNull(reflectedType, nameof(reflectedType));
```

Comment:
True. I'm going to just remove the `reflectedType` null check here as uses of it guard against null (but don't throw).